### ISSUE TITLE
impossible to start a stopped container

### ISSUE THREAD

**Issue by colundrum** [containers/libpod#4790]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

**Steps to reproduce the issue:**

1. stop container 1: `podman stop drone`

2. stop container 2: `podman stop bitwarden`

3. start again each container: `podman start drone && podman start bitwarden`

**Describe the results you received:**

```
Error: unable to start container "unruffled_goldberg": container_linux.go:338: creating new parent process caused "container_linux.go:1913: running lstat on namespace path \"/run/user/1000/netns/cni-eec32aae-bd8a-abb5-5be8-0e560fd6b4a5\" caused \"lstat /run/user/1000/netns/cni-eec32aae-bd8a-abb5-5be8-0e560fd6b4a5: no such file or directory\"": OCI runtime command not found error

Error: unable to start container "drone": container_linux.go:338: creating new parent process caused "container_linux.go:1913: running lstat on namespace path \"/run/user/1000/netns/cni-d6d06d6e-8553-812f-bcfe-8c4019f6d68e\" caused \"lstat /run/user/1000/netns/cni-d6d06d6e-8553-812f-bcfe-8c4019f6d68e: no such file or directory\"": OCI runtime command not found error
```

**Describe the results you expected:**

I expected that podman restart successfully each container

**Additional information you deem important (e.g. issue happens only occasionally):**

This happen each time, it's impossible to stop and after start again a container, I need to rm and re-create it

**Output of `podman version`:**

```
Version:            1.6.2
RemoteAPI Version:  1
Go Version:         go1.10.4
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  git commit: ""
  go version: go1.10.4
  podman version: 1.6.2
host:
  BuildahVersion: 1.11.3
  CgroupVersion: v1
  Conmon:
    package: 'conmon: /usr/bin/conmon'
    path: /usr/bin/conmon
    version: 'conmon version 2.0.3, commit: unknown'
  Distribution:
    distribution: debian
    version: "10"
  IDMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  MemFree: 12877807616
  MemTotal: 33642573824
  OCIRuntime:
    name: runc
    package: 'cri-o-runc: /usr/lib/cri-o-runc/sbin/runc'
    path: /usr/lib/cri-o-runc/sbin/runc
    version: 'runc version spec: 1.0.1-dev'
  SwapFree: 17599291392
  SwapTotal: 17599291392
  arch: amd64
  cpus: 12
  eventlogger: journald
  hostname: sofia
  kernel: 5.3.0-0.bpo.2-amd64
  os: linux
  rootless: true
  slirp4netns:
    Executable: /usr/bin/slirp4netns
    Package: 'slirp4netns: /usr/bin/slirp4netns'
    Version: |-
      slirp4netns version 0.4.2
      commit: unknown
  uptime: 271h 41m 26.42s (Approximately 11.29 days)
registries:
  blocked: null
  insecure: null
  search:
  - docker.io
  - registry.fedoraproject.org
  - registry.access.redhat.com
store:
  ConfigFile: /home/colundrum/.config/containers/storage.conf
  ContainerStore:
    number: 3
  GraphDriverName: vfs
  GraphOptions: {}
  GraphRoot: /home/colundrum/.local/share/containers/storage
  GraphStatus: {}
  ImageStore:
    number: 6
  RunRoot: /run/user/1000
  VolumePath: /home/colundrum/.local/share/containers/storage/volumes
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman/bionic,now 1.6.2-1~ubuntu18.04~ppa1 amd64 [installed]
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

Physical server
Distributor ID: Debian
Description:    Debian GNU/Linux 10 (buster)
Release:        10
Codename:       buster

Repository used: ```deb http://ppa.launchpad.net/projectatomic/ppa/ubuntu bionic main```

**Comment by colundrum** [comment]
After an update of podman to 1.6.2-1, I obtain:

```
Error: unable to start container "drone": container create failed (no logs from conmon): EOF
```

When I try to start with the same command line `podman start bitwarden`

**Comment by mheon** [comment]
Any details on the two containers in question - what commands were used to create them, anything unusual about them?

Can you add `--log-level=debug` to your `podman start` commands and provide the output, as well as anything in syslog from `conmon`?

**Comment by colundrum** [comment]
Sorry, I'll recreated containers but I'll do the same in test env, tomorrow, using same flags

Nothing about conmon in logs

```
podman run -d \
        --restart always \
        --name bitwarden \
        -v /var/lib/bitwardenrs-data/:/data \
        -e DOMAIN=https://vault.domain.tld \
        -e ADMIN_TOKEN=MY_ADMIN_TOKEN \
        -e WEBSOCKET_ENABLED=true \
        -p 127.0.0.1:3012:3012 \
        -e ROCKET_PORT=9001 \
        -p 127.0.0.1:9001:9001 \
        bitwardenrs/server:latest
```

```
podman run -d \
        -v /var/lib/drone:/data \
        -e DRONE_GIT_ALWAYS_AUTH=false \
        -e DRONE_GITEA_SERVER=https://git.domain.tld \
        -e DRONE_GITEA_CLIENT_ID=MY_UUID \
        -e DRONE_GITEA_CLIENT_SECRET=MY_SECRET \
        -e DRONE_SERVER_HOST=drone.domain.tld \
        -e DRONE_SERVER_PROTO=https \
        -e DRONE_TLS_AUTOCERT=false \
        -e DRONE_RPC_SECRET=MY_RPC_SECRET \
        -e DRONE_AGENTS_ENABLED=true \
        -e DRONE_CRON_INTERVAL=1m \
        -e DRONE_DATABASE_SECRET=MY_DB_PASSWORD \
        -e DRONE_USER_CREATE=username:my_username,admin:true \
        -e DRONE_USER_FILTER=my_username \
        -e DRONE_ADMIN=my_username \
        -p 127.0.0.1:20080:80 \
        -p 127.0.0.1:20443:443 \
        --restart=always \
        --name=drone \
        drone/drone
```

**Comment by colundrum** [comment]
```
time="2020-01-06T16:49:50+01:00" level=debug msg="using conmon: \"/usr/libexec/podman/conmon\""
time="2020-01-06T16:49:50+01:00" level=debug msg="Initializing boltdb state at /home/colundrum/.local/share/containers/storage/libpod/bolt_state.db"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using graph driver vfs"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using graph root /home/colundrum/.local/share/containers/storage"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using run root /run/user/1000"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using static dir /home/colundrum/.local/share/containers/storage/libpod"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using tmp dir /run/user/1000/libpod/tmp"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using volume path /home/colundrum/.local/share/containers/storage/volumes"
time="2020-01-06T16:49:50+01:00" level=debug msg="Set libpod namespace to \"\""
time="2020-01-06T16:49:50+01:00" level=debug msg="[graphdriver] trying provided driver \"vfs\""
time="2020-01-06T16:49:50+01:00" level=debug msg="Initializing event backend journald"
time="2020-01-06T16:49:50+01:00" level=debug msg="using runtime \"/usr/lib/cri-o-runc/sbin/runc\""
time="2020-01-06T16:49:50+01:00" level=warning msg="Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument"
time="2020-01-06T16:49:50+01:00" level=info msg="running as rootless"
time="2020-01-06T16:49:50+01:00" level=debug msg="using conmon: \"/usr/libexec/podman/conmon\""
time="2020-01-06T16:49:50+01:00" level=debug msg="Initializing boltdb state at /home/colundrum/.local/share/containers/storage/libpod/bolt_state.db"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using graph driver vfs"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using graph root /home/colundrum/.local/share/containers/storage"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using run root /run/user/1000"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using static dir /home/colundrum/.local/share/containers/storage/libpod"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using tmp dir /run/user/1000/libpod/tmp"
time="2020-01-06T16:49:50+01:00" level=debug msg="Using volume path /home/colundrum/.local/share/containers/storage/volumes"
time="2020-01-06T16:49:50+01:00" level=debug msg="Set libpod namespace to \"\""
time="2020-01-06T16:49:50+01:00" level=debug msg="Initializing event backend journald"
time="2020-01-06T16:49:50+01:00" level=debug msg="using runtime \"/usr/lib/cri-o-runc/sbin/runc\""
time="2020-01-06T16:49:50+01:00" level=warning msg="Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument"
time="2020-01-06T16:49:50+01:00" level=debug msg="Made network namespace at /run/user/1000/netns/cni-a4c5018b-323e-2254-7b89-5ed38380410b for container 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd"
time="2020-01-06T16:49:50+01:00" level=debug msg="mounted container \"7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd\" at \"/home/colundrum/.local/share/containers/storage/vfs/dir/8d0f8ef755dd6fb834f3bbc7397f2a5a109677ee2e7bb8bedb7179a7b6e26a69\""
time="2020-01-06T16:49:50+01:00" level=debug msg="slirp4netns command: /usr/bin/slirp4netns --api-socket /run/user/1000/libpod/tmp/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd.net --disable-host-loopback --mtu 65520 --enable-sandbox -c -e 3 -r 4 --netns-type=path /run/user/1000/netns/cni-a4c5018b-323e-2254-7b89-5ed38380410b tap0"
time="2020-01-06T16:49:50+01:00" level=debug msg="Created root filesystem for container 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd at /home/colundrum/.local/share/containers/storage/vfs/dir/8d0f8ef755dd6fb834f3bbc7397f2a5a109677ee2e7bb8bedb7179a7b6e26a69"
time="2020-01-06T16:49:50+01:00" level=debug msg="/etc/system-fips does not exist on host, not mounting FIPS mode secret"
time="2020-01-06T16:49:50+01:00" level=debug msg="Created OCI spec for container 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd at /home/colundrum/.local/share/containers/storage/vfs-containers/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/userdata/config.json"
time="2020-01-06T16:49:50+01:00" level=debug msg="/usr/libexec/podman/conmon messages will be logged to syslog"
time="2020-01-06T16:49:50+01:00" level=debug msg="running conmon: /usr/libexec/podman/conmon" args="[--api-version 1 -c 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd -u 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd -r /usr/lib/cri-o-runc/sbin/runc -b /home/colundrum/.local/share/containers/storage/vfs-containers/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/userdata -p /run/user/1000/vfs-containers/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/userdata/pidfile -l k8s-file:/home/colundrum/.local/share/containers/storage/vfs-containers/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/userdata/ctr.log --exit-dir /run/user/1000/libpod/tmp/exits --socket-dir-path /run/user/1000/libpod/tmp/socket --log-level debug --syslog --conmon-pidfile /run/user/1000/vfs-containers/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/userdata/conmon.pid --exit-command /usr/bin/podman --exit-command-arg --root --exit-command-arg /home/colundrum/.local/share/containers/storage --exit-command-arg --runroot --exit-command-arg /run/user/1000 --exit-command-arg --log-level --exit-command-arg error --exit-command-arg --cgroup-manager --exit-command-arg cgroupfs --exit-command-arg --tmpdir --exit-command-arg /run/user/1000/libpod/tmp --exit-command-arg --runtime --exit-command-arg runc --exit-command-arg --storage-driver --exit-command-arg vfs --exit-command-arg --events-backend --exit-command-arg journald --exit-command-arg container --exit-command-arg cleanup --exit-command-arg 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd]"
[conmon:d]: failed to write to /proc/self/oom_score_adj: Permission denied

time="2020-01-06T16:49:50+01:00" level=debug msg="Cleaning up container 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd"
time="2020-01-06T16:49:50+01:00" level=debug msg="Tearing down network namespace at /run/user/1000/netns/cni-a4c5018b-323e-2254-7b89-5ed38380410b for container 7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd"
time="2020-01-06T16:49:50+01:00" level=debug msg="unmounted container \"7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd\""
time="2020-01-06T16:49:50+01:00" level=error msg="unable to start container \"bitwarden\": container create failed (no logs from conmon): EOF"
```

**Comment by mheon** [comment]
After the `--log-level=debug` run you should now have Conmon logs in syslog, can you provide those? The underlying error should be reported there

**Comment by colundrum** [comment]
```
8232:Jan  6 16:49:38 sofia conmon: conmon 7d055e16ec437c397a22 <ndebug>: failed to write to /proc/self/oom_score_adj: Permission denied#012
8233:Jan  6 16:49:38 sofia conmon: conmon 7d055e16ec437c397a22 <ninfo>: attach sock path: /run/user/1000/libpod/tmp/socket/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/attach
8234:Jan  6 16:49:38 sofia conmon: conmon 7d055e16ec437c397a22 <ninfo>: addr{sun_family=AF_UNIX, sun_path=/run/user/1000/libpod/tmp/socket/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/attach}
8235:Jan  6 16:49:38 sofia conmon: conmon 7d055e16ec437c397a22 <ninfo>: terminal_ctrl_fd: 13
8236:Jan  6 16:49:38 sofia conmon: conmon 7d055e16ec437c397a22 <error>: Failed to mkfifo at /home/colundrum/.local/share/containers/storage/vfs-containers/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/userdata/winsz: File exists
8240:Jan  6 16:49:50 sofia conmon: conmon 7d055e16ec437c397a22 <ndebug>: failed to write to /proc/self/oom_score_adj: Permission denied#012
8241:Jan  6 16:49:50 sofia conmon: conmon 7d055e16ec437c397a22 <ninfo>: attach sock path: /run/user/1000/libpod/tmp/socket/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/attach
8242:Jan  6 16:49:50 sofia conmon: conmon 7d055e16ec437c397a22 <ninfo>: addr{sun_family=AF_UNIX, sun_path=/run/user/1000/libpod/tmp/socket/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/attach}
8243:Jan  6 16:49:50 sofia conmon: conmon 7d055e16ec437c397a22 <ninfo>: terminal_ctrl_fd: 13
8244:Jan  6 16:49:50 sofia conmon: conmon 7d055e16ec437c397a22 <error>: Failed to mkfifo at /home/colundrum/.local/share/containers/storage/vfs-containers/7d055e16ec437c397a22051c791c0cb422ee2a945b68c470a5d956687afaeccd/userdata/winsz: File exists
```

**Comment by haircommander** [comment]
ah, version 1.6.2 of podman is a bit out of date for conmon 2.0.3. We need podman 1.6.4 for it to work seemlessly. A temp work around that isn't great is you can manually delete that winsz file between stops and starts.

**Comment by colundrum** [comment]
@haircommander but I use ppa of projectatomic … no new version.
Isn't my fault if this ppa isn't up-to-date with correct version between packages

**Comment by haircommander** [comment]
you are very correct, and thanks for bringing our attention to it :smile: @lsm5 @jnovy PTAL

**Comment by ParitoshBh** [comment]
Any updates on this? I am facing the exact same issue and ppa still seems to be carrying version 1.6.2. Also noticed there's a [snap version of podman](https://snapcraft.io/podman), would that be better to use (seeing that it was updated recently)?

**Comment by mheon** [comment]
I believe that @lsm5 has been making some packaging changes to the PPA related to using obs; @lsm5 any updates on getting 1.7.0 built there?

**Comment by colundrum** [comment]
@ParitoshBh Snap is like LXC so it's not a solution when you have dedicated server to run podman
In that case, why not running podman in VM using Fedora ?

@mheon Are you referring to [Open Build Service](https://openbuildservice.org/) ? Have you any ETA about when it'll be okay for prod because deleting good container to after restart them, it's a little overkill and bad experience for user "pro-Docker" when I force migration to podman.

**Comment by rhatdan** [comment]
PTAL https://github.com/containers/libpod/pull/4873

**Comment by colundrum** [comment]
@rhatdan Thanks
@ParitoshBh you can look directly at: https://github.com/containers/libpod/blob/d9bcf20b3de0f2f085044b1724154ba1950d46d6/install.md#debian-1

**Comment by ParitoshBh** [comment]
@colundrum Thanks. Followed the instructions, can confirm this particular issue has been resolved (version 1.7.0-rc1).

**Comment by colundrum** [comment]
@ParitoshBh for me, I used stable and not testing so I've podman 1.6.4~2 and all fine
@rhatdan thanks
